crates/bevy_transform/src/components/transform.rs

```diff
@@ -527,35 +533,42 @@ impl Default for Transform {
     }
 }
 
-/// The transform is expected to be non-degenerate and without shearing, or the output
-/// will be invalid.
-impl From<GlobalTransform> for Transform {
-    fn from(transform: GlobalTransform) -> Self {
-        transform.compute_transform()
+#[cfg(feature = "bevy-support")]
+mod bevy_support {
+    use std::ops::Mul;
+
+    use super::*;
+
+    /// The transform is expected to be non-degenerate and without shearing, or the output
+    /// will be invalid.
+    impl From<GlobalTransform> for Transform {
```

Comment:
This shouldn't be feature-gated.

Reply:
Currently `GlobalTransform` is feature gated in this PR since it has little use without the surrounding machinery.

I moved the math related impls out of the feature gate here but kept `GlobalTransform` because of the above, thoughts?

Reply:
Hmm. I think that we should probably expose `GlobalTransform` (and other future transform types) to all users: it's not adding any dependencies and otherwise is very little code.

Otherwise, if a user *does* want it, there's no way to get it without the rest of Bevy, replicating the original problem.
It's also a nice simple rule to enforce as we maintain the crate: things that need bevy_ecs stay out, everything else stays in.

Reply:
Added back GlobalTransform